tutorials/3d/procedural_geometry/arraymesh.rst

```diff
@@ -126,6 +142,24 @@ by adding each array to ``surface_array`` and then committing to the mesh.
 
     mesh.add_surface_from_arrays(Mesh.PRIMITIVE_TRIANGLES, surface_array) # No blendshapes, lods, or compression used.
 
+ .. code-tab:: csharp C#
+
+    var verts = vertList.ToArray();
+    var uvs = uvList.ToArray();
+    var normals = normalList.ToArray();
+    var indices = indexList.ToArray();
+    
+    surfaceArray[(int)Mesh.ArrayType.Vertex] = verts;
+    surfaceArray[(int)Mesh.ArrayType.TexUV] = uvs;
+    surfaceArray[(int)Mesh.ArrayType.Normal] = normals;
+    surfaceArray[(int)Mesh.ArrayType.Index] = indices;
+
+    var arrMesh = Mesh as ArrayMesh;
+    if (arrMesh != null)
+    {
+        arrMesh.AddSurfaceFromArrays(Mesh.PrimitiveType.Triangles, surfaceArray); // No blendshapes, lods, or compression used.
```

Comment:
This is creating horizontal scroll which we tend to avoid, maybe put the comment on its own line (and do the same in GDScript).

Reply:
Thanks, fixed for C# and GDScript both.